pkg/instancegroups/rollingupdate_test.go

```diff
@@ -667,3 +668,5 @@ func TestRollingUpdateUnknownRole(t *testing.T) {
 		}
 	}
 }
+
+*/
```

Comment:
Hmmm... not wild about disabling the test coverage we do have for this...

Reply:
lol :-) I will put it back .. i promise, just wanted to seal the deal on the implementation / premise before updating the tests